README.md

```diff
@@ -28,6 +28,7 @@ Second argument of NewClient function is an object that implements
 [http.RoundTripper](http://golang.org/pkg/net/http/#RoundTripper)
 interface, it can be used to get more control over connection options.
 By default it initialized by http.DefaultTransport object.
+These tags support the omitempty property.
```

Comment:
This doesn't seem like the right place for this comment.

Reply:
I'm all for changing it to wherever seems more appropriate. tbh, I had written the change, then realized I did not mention it in the readme.

Reply:
Move it under the "Structs decoded to struct by following rules:" heading. Also fix that heading to say "Structs encoded to struct by following rules:"